doc/api/errors.md

```diff
@@ -2412,6 +2412,12 @@ The `--entry-type=...` flag is not compatible with the Node.js REPL.
 Used when an [ES Module][] loader hook specifies `format: 'dynamic'` but does
 not provide a `dynamicInstantiate` hook.
 
+<a id="ERR_FEATURE_UNAVAILABLE_ON_PLATFORM"></a>
+#### ERR_FEATURE_UNAVAILABLE_ON_PLATFORM
```

Comment:
```suggestion
#### `ERR_FEATURE_UNAVAILABLE_ON_PLATFORM`
```
To fix the lint error after https://github.com/nodejs/node/pull/31086 was merged.